### PR TITLE
[timeseries] Update Chronos-2 tutorial

### PR DESCRIPTION
**Pull request by abdulfatir**
*Issue #, if available:*

*Description of changes:*

☑️  TODOs:

- [x] Add some details on "why use Chronos-2 in AG?"
- [x] Ensemble of Chronos-2 ZS and FT (?)
- [x] Backtest predictions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/timeseries/forecasting-chronos.ipynb [hunk 1]
```diff
@@ -540,240 +892,203 @@
...
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "data = TimeSeriesDataFrame.from_path(\n",
-    "    \"https://autogluon.s3.amazonaws.com/datasets/timeseries/grocery_sales/test.csv\",\n",
-    ")\n",
-    "data.head()"
+    "predictor.feature_importance(test_data, model=\"Chronos2\", relative_scores=True)"
```
Comment (shchur) [inline docs/tutorials/timeseries/forecasting-chronos.ipynb:952]: Could you mention in the next cell that `relative_score=True` => `using this feature leads to ~32% error reduction on the test set`.

File: docs/tutorials/timeseries/forecasting-chronos.ipynb [hunk 2]
```diff
@@ -871,30 +1186,54 @@
...
+    "\n",
+    "#### Does fine-tuning always improve Chronos-2's forecasting accuracy?\n",
+    "\n",
+    "Fine-tuning a foundation model like Chronos-2 involves many hyperparameter choices. AG-TS provides reasonable defaults that performed well in large-scale benchmarking, but they may not be optimal for every use case. We recommend fine-tuning only when you have a reasonable number of time series and sufficient historical data, as limited data can lead to overfitting or degraded performance. If you observe degraded accuracy, we recommend increasing the size of the training data and experimenting with different fine-tuning hyperparameters.\n",
```
Comment (shchur) [inline docs/tutorials/timeseries/forecasting-chronos.ipynb]: What are reasonable values here? (e.g. at least X time series / at least Y observations in total)

File: docs/tutorials/timeseries/forecasting-chronos.ipynb [hunk 3]
```diff
@@ -871,30 +1186,54 @@
...
+    "    ...,\n",
+    "    hyperparameters={\"Chronos2\": {\"cross_learning\": False}},\n",
+    ")\n",
+    "```\n",
     "\n",
     "#### Where can I ask specific questions on Chronos?\n",
     "\n",
-    "The AutoGluon team are among the core developers of Chronos. So you can ask Chronos-related questions on AutoGluon channels such \n",
+    "Members of the AutoGluon team are among the core developers of Chronos. So you can ask Chronos-related questions on AutoGluon channels such \n",
     "as the Discord [server](https://discord.gg/wjUmjqAc2N), or [GitHub](https://github.com/autogluon/autogluon). You can also join \n",
```
Comment (shchur) [inline docs/tutorials/timeseries/forecasting-chronos.ipynb]: Maybe we can remove the Discord server here? We are a lot less active there compared to GitHub

File: docs/tutorials/timeseries/forecasting-chronos.ipynb [hunk 4]
```diff
@@ -279,132 +485,289 @@
...
-    "    time_limit=60,  # time limit in seconds\n",
+    "predictor = TimeSeriesPredictor(\n",
+    "    prediction_length=prediction_length,\n",
+    "    target=\"load\",\n",
+    "    known_covariates_names=[\"airtemperature\", \"dewtemperature\", \"sealvlpressure\"],\n",
+    "    eval_metric=\"MASE\",\n",
+    ").fit(\n",
+    "    train_data,\n",
+    "    hyperparameters={\"Chronos\": {}, \"Chronos2\": {}},\n",
     "    enable_ensemble=False,\n",
```
Comment (shchur) [inline docs/tutorials/timeseries/forecasting-chronos.ipynb:762]: If the ensemble has a better test score, should we enable it?

**Review by shchur — APPROVED** [review]
Thanks, looks great! Just a few optional comments

File: docs/tutorials/timeseries/forecasting-chronos.ipynb (hunk 2 above)
Comment (abdulfatir) [reply docs/tutorials/timeseries/forecasting-chronos.ipynb]: Added: (e.g., >100 time series with a median history length larger than `3 * prediction_length`).

However, this is based mostly on vibes.

File: docs/tutorials/timeseries/forecasting-chronos.ipynb (hunk 4 above)
Comment (abdulfatir) [reply docs/tutorials/timeseries/forecasting-chronos.ipynb:762]: In this case, it does not.

File: docs/tutorials/timeseries/forecasting-chronos.ipynb (hunk 3 above)
Comment (abdulfatir) [reply docs/tutorials/timeseries/forecasting-chronos.ipynb]: Done.

File: docs/tutorials/timeseries/forecasting-chronos.ipynb (hunk 2 above)
Comment (shchur) [reply docs/tutorials/timeseries/forecasting-chronos.ipynb]: That's fine

**Comment by abhishek-iitmadras** [comment]
Hi @abdulfatir @shchur 

Is Chronos-2 going to replace Chronos-Bolt as the default ts model?

**Comment by abhishek-iitmadras** [comment]
> Hi @abdulfatir @shchur
> 
> Is Chronos-2 going to replace Chronos-Bolt as the default ts model?


Ok i guess after this patch https://github.com/autogluon/autogluon/pull/5480 , 
And after autogluon 1.5 release, am i right ?
